.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release
+on: [workflow_dispatch]
+
+jobs:
+  bumpversion:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.tag_version.outputs.new_version }}
+      previous_tag: ${{ steps.tag_version.outputs.previous_tag }}
+      bump_commit_sha: ${{ steps.bumpversion.outputs.commit_hash }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get next version
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          default_bump: false
+          default_prerelease_bump: false
+          dry_run: true
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Create bumpversion
+        if: steps.tag_version.outputs.new_version
+        run: |
+          pip install bumpversion
```

Comment:
I was reading the bumpversion [documentation](https://github.com/peritus/bumpversion) and They recommend using one of the maintained fork, such as bump2version. For this case, the only change required would be `pip install bump2version`

Reply:
Let's do it! Thanks